### ISSUE TITLE
Proposal for legacy terms in epilepsy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4295]
The epilepsy community has updated its definitions and system for describing seizures.
https://www.ilae.org/guidelines/definition-and-classification
In some cases, the concepts are just different, so we cannot just update labels. In other cases we can update labels.
In discussions today with eh epilepsiome group, we think that having a "legacy" category:
seizures
--pre-ILAE2017 terms (workign on a better name)
----legacy term 1
----legacy term 2
etc

We would expand the definitions of the legacy terms with recommendations of what new terms to use. 
We would say please do not use the terms for new data
We would consider saying that the terms would be obsoleted in 2(?) years

@drseb @mellybelly @cmungall thoughts?

**Comment by cmungall** [comment]
I assume that immediate obsoletion would cause too much churn.

I think the above is fine. Of course the grouping is not really a proper class, it's more of a metaclass, but I think it's OK. I might also define a subset for these.

**Comment by drseb** [comment]
where would you put the pre-ILAE-class then? as a new root?

**Comment by pnrobinson** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney

These are the terms that are "left over". We should discuss whether to keep them or put them into a legacy category
a) Multifocal seizures HP:0031165
https://hpo.jax.org/app/browse/term/HP:0031165
b) Nocturnal seizures HP:0031951
https://hpo.jax.org/app/browse/term/HP:0031951
c) Symptomatic seizures HP:0011145
https://hpo.jax.org/app/browse/term/HP:0011145

(and their subterms in one case).

**Comment by GBalagura** [comment]
@DLewisSmith 
@galerp 
@shiva-g
@ihelbig
@hughkearney
@pnrobinson 

**Nocturnal seizures HP:0031951**
Don’t like that. BUT I suggest to think of tese possibilities:
a.	could it be useful to keep it in case we want to underline that the seizures are ONLY occurring during sleep and not awake (anymore)? In that case, I would change it to “DURING SLEEP” and add some child terms for the different phases of sleep, because that makes a difference as some are more sensitive to seizures. 
b.	Same as above, but indicating it as “modifier” to be used with, instead of a “phenotype” in the seizure ontology
c.	Just obsolete

**Comment by GBalagura** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@hughkearney
@pnrobinson

**Multifocal seizures HP:0031165**
I don’t agree with replacing “multifocal” with “migrating”, because <<Migrating>> seizures have usually a pretty definite EEG correlate, with Focal seizures arising independently in both hemispheres, that migrate from one cortical region to another randomly but consecutively in the same seizure (https://www.epilepsydiagnosis.org/syndrome/infancy-migrating-focal-videos.html) (McTague et al, Brain, 2013). So usually they don’t overlap. 
Whereas, <<multifocal>> seizures are more a series of multiple different seizures starting from different networks (Operational classification of seizure types by the International League Against Epilepsy: Position Paper of the ILAE Commission for Classification and Terminology, 2017), not requiring consecutiveness for their definition, because they actually can overlap and don’t show a pattern. 
[This is especially true in neonatal seizures, where we cannot speak about generalization, but seizures involving multiple sites together can be defined multifocal (but I don’t want to confuse things).]
But, I don’t like “multifocal” because it encourages lazy coding and is not really "prescribed" by ILAE2017. I am thinking how to make it right.

**Comment by pnrobinson** [comment]
The use case for Nocturnal seizure is the fact that some diseases can be characterized by this, see e.g. 
https://omim.org/entry/604364
The HPO currently has 4 such annotations:
https://hpo.jax.org/app/browse/term/HP:0031951

**Comment by DLewisSmith** [comment]
They are simply very common. Most people with frequent seizures have them in sleep. Rare individuals only have them in sleep. Transitioning between sleep stages seems to precipitate seizures. Frontal epilepsies have traditionally been considered to be very nocturnal and nocturnal seizures often seem to cluster with several in one night.

As Anna pointed out, there is a tension between describing this pattern of seizures (and resulting syndromes) as nocturnal, sleep-related, or even ‘on awakening’.

**Comment by GBalagura** [comment]
> The use case for Nocturnal seizure is the fact that some diseases can be characterized by this, see e.g.
> https://omim.org/entry/604364
> The HPO currently has 4 such annotations:
> https://hpo.jax.org/app/browse/term/HP:0031951

I agree, they are too common, it is hard to tell that some are really characterized by that. 
The only thing I could consider is e.g. in some children with Dravet (my patients), but also other disease, seizures were frequent when patient was awake at the beginning of the disease, but became "only during sleep" later in life... I don't know if this pattern can be of any interest for a longitudinal deep phenotyping. Maybe yes. But again, I wouldn't call it "nocturnal" but "sleep-related" or smtg, and specify if occurring at awakening, when falling asleep, or in other stages (if long term eeg available, for example). But then the problem would be: and what about awake seizures? Assume that if sleep-related is not indicated, then it's awake?

Also, a bit off-topic:  it would be interesting to think about how would "sleep-deprivation" fit into these. Probably it wouldn't, because we don't have enough evidence of lower seizure threshold in people without epilepsy... but worth a thought.

**Comment by DLewisSmith** [comment]
Very true. We see a lot of adults whose severe paediatric epilepsy syndrome goes from a global diurnal pattern to a nocturnal pattern.

**Comment by GBalagura** [comment]
I will draft a small sub-ontology for that as well and send it to you to discuss, then.

**Comment by pnrobinson** [comment]
I think we were able to refactor everything without resorting to this category!
